### PR TITLE
Check all candidates of module which matches class name.

### PR DESCRIPTION
**Pull request by kateinoigakukun**
Fix https://github.com/IBDecodable/IBLinter/issues/113

### CODE REVIEW

File: Sources/IBLinterKit/Rules/CustomModuleRule.swift [hunk 1]
```diff
@@ -70,41 +70,31 @@ extension Rules {
...
-            guard let expectedModule = moduleClasses.first(where: { $0.value.contains(customClass) }) else {
-                return []
-            }
-            let message = "It does not match custom module rule in \(fileNameWithoutExtension). Custom module of \(customClass) is \(expectedModule.key)"
-            let violation = Violation(pathString: file.pathString, message: message, level: .error)
-            guard let customModule = classableObject.customModule, expectedModule.key == customModule else {
+            let moduleCandidates = moduleClasses.lazy.filter({ $0.value.contains(customClass) }).map { $0.key }
+            guard !moduleCandidates.isEmpty else { return [] }
+            guard let customModule = classableObject.customModule, moduleCandidates.contains(customModule) else {
+                let message = "It does not match custom module rule in \(fileNameWithoutExtension). Custom module of \(customClass) is one of \(moduleCandidates)"
```
Comment (sidepelican) [inline Sources/IBLinterKit/Rules/CustomModuleRule.swift]:
```swift
\(moduleCandidates)
```

this outputs like this

```
LazyMapSequence<LazyFilterSequence<Dictionary<String, Array<String>>>, String>(_base: Swift.LazyFilterSequence<Swift.Dictionary<Swift.String, Swift.Array<Swift.String>>>(_base: ["ModuleA": ["AView", "BView", "CView",..... (a lot of classes)
```

I expects module name lists.

Comment (kateinoigakukun) [reply Sources/IBLinterKit/Rules/CustomModuleRule.swift]: I updated the message format.

**Comment by sidepelican** [comment]
LGTM! It works. Thank you!

(After this implementation, it found that our project has another misuse of `.xib` 😅)

**Comment by giginet** [comment]
@sidepelican Aren't you interested to be committer? 😆

**Pull request merged**